lightningd/opening_control.c

```diff
@@ -780,6 +781,7 @@ static void json_fund_channel(struct command *cmd,
 		   p_req("id", json_tok_pubkey, &id),
 		   p_req("satoshi", json_tok_tok, &sattok),
 		   p_opt("feerate", json_tok_feerate, &feerate_per_kw),
+		   p_opt_def("private", json_tok_bool, &private_channel, false),
```

Comment:
The spec calls this bit `announce_channel`; should we invert this and call it "announce", default=true?

Reply:
works for me. :+1: